### ISSUE TITLE
Nothing patched on Mac

### ISSUE THREAD

**Issue by Shinigami92** [nonara/ts-patch#4]
_After applying #3_ 

```shell
 ~/project yarn ts-patch check
yarn run v1.21.1
$ /project/node_modules/.bin/ts-patch check
Checking TypeScript v3.7.3 installation in /project/node_modules/typescript

[-] tsc.js is not patched.
[-] tsserverlibrary.js is not patched.
[-] typescript.js is not patched.
[-] typescriptServices.js is not patched.
✨  Done in 0.47s.


 ~/project yarn ts-patch install --persist
yarn run v1.21.1
$ /project/node_modules/.bin/ts-patch install --persist
[+] ts-patch installed!
[+] Enabled persistence for /project/node_modules/typescript
✨  Done in 91.87s.


 ~/project yarn ts-patch check            
yarn run v1.21.1
$ /project/node_modules/.bin/ts-patch check
Checking TypeScript v3.7.3 installation in /project/node_modules/typescript

[-] tsc.js is not patched.
[-] tsserverlibrary.js is not patched.
[-] typescript.js is not patched.
[-] typescriptServices.js is not patched.
✨  Done in 0.72s.
```

**Comment by nonara** [comment]
Interesting. Thanks for letting me know. Would you mind running patch with --verbose flag and posting the output?

Also, would you mind opening one of the reportedly patched js files with notepad++ and looking at the bottom to see if tsPatch logic is there?

You should see something along these lines near the bottom:
```TypeScript
    Object.assign(ts, {
        tspVersion: tspVersion,
        PluginCreator: PluginCreator,
        originalCreateProgram: ts.createProgram,
        createProgram: createProgram
    });
```

**Comment by Shinigami92** [comment]
Is this quantum technology?!?!?

```shell
~/project yarn ts-patch install --persist --verbose
yarn run v1.21.1
$ /project/node_modules/.bin/ts-patch install --persist --verbose
[~] Patching tsc.js in /project/node_modules/typescript/lib
[+] Successfully patched tsc.js.

[~] Patching tsserverlibrary.js in /project/node_modules/typescript/lib
[+] Successfully patched tsserverlibrary.js.

[~] Patching typescript.js in /project/node_modules/typescript/lib
[+] Successfully patched typescript.js.

[~] Patching typescriptServices.js in /project/node_modules/typescript/lib
[+] Successfully patched typescriptServices.js.

[+] ts-patch installed!
[~] Enabling persistence in /project/node_modules/typescript
[+] Enabled persistence for /project/node_modules/typescript
✨  Done in 1.03s.
 ~/project yarn ts-patch check --verbose            
yarn run v1.21.1
$ /project/node_modules/.bin/ts-patch check --verbose
Checking TypeScript v3.7.3 installation in /project/node_modules/typescript

[+] tsc.js is patched with ts-patch version 1.0.4.

[+] tsserverlibrary.js is patched with ts-patch version 1.0.4.

[+] typescript.js is patched with ts-patch version 1.0.4.

[+] typescriptServices.js is patched with ts-patch version 1.0.4.

✨  Done in 0.44s.
 ~/projeect yarn ts-patch check          
yarn run v1.21.1
$ /project/node_modules/.bin/ts-patch check
Checking TypeScript v3.7.3 installation in /project/node_modules/typescript

[+] tsc.js is patched with ts-patch version 1.0.4.
[+] tsserverlibrary.js is patched with ts-patch version 1.0.4.
[+] typescript.js is patched with ts-patch version 1.0.4.
[+] typescriptServices.js is patched with ts-patch version 1.0.4.
✨  Done in 0.42s.
```
